paco-classifier-service/main.py

```
@@ -0,0 +1,123 @@
+    masked[mask == 0] = (255, 255, 255)
+    alpha_channel = np.ones(mask.shape, dtype=mask.dtype) * 255
+    alpha_channel[mask == 0] = 0
+    b, g, r = cv2.split(masked)
+    rgba = cv2.merge((b, g, r, alpha_channel))
+    ok, buf = cv2.imencode(".png", rgba)
+    if not ok:
+        raise RuntimeError("PNG encode failed")
+    return buf.tobytes()
+
+@app.post("/classify")
+async def classify(image: UploadFile = File(...)):
```

Ruff (0.16.1)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)